code/modules/antagonists/traitor/contractor/datums/objective_contract.dm

```diff
@@ -129,6 +139,7 @@
 			"Port Primary Hallway",
 			"Quartermaster's Office",
 			"Research Director's Office",
+			"Research Division",
 			"Research and Development",
```

Comment:
Undocumented change? This adds the main science hallway to the list too.

Reply:
It should not, "Research Division" is used for random maint areas, its not the actual research department AFAIK

Reply:
Ah, I just looked deeper and yes, for some random reason this IS used as the primary hallway for Science, despite the area path being `/area/medical/research`.

I'll remove this.

Reply:
1. Research division already exists inside medium
![image](https://user-images.githubusercontent.com/25063394/128485710-6cf99560-7e7c-416a-a6ae-9178d0dc1897.png)

2. Research division is the entirety of science
```dm
/area/medical/research
	name = "Research Division"
	icon_state = "research"
```

![image](https://user-images.githubusercontent.com/25063394/128486022-e3f8b2bc-467c-44cd-b6a5-d048e0441227.png)

Remove it from here, its already in medium